src/languages/ruby.md

```diff
@@ -4,8 +4,8 @@ Platform.sh supports deploying any Ruby application. Your application can use
 any Ruby application server such as Unicorn or Puma and deploying a Rails or
 a Sinatra app is very straight forward.
 
-### Supported versions
-#### Ruby MRI
+## Supported versions
```

Comment:
I'm not totally convinced (yet?) that there's anything wrong with skipping header levels, but if that's a problem, then I think we should fix that without removing the mention of Ruby MRI: the one extra header is concise enough that it should be worth it to leave it in there.